ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/LMDB.hs

```diff
@@ -18,6 +18,9 @@ module Ouroboros.Consensus.Storage.LedgerDB.HD.LMDB
   , LMDBLimits
   , TraceDb(..)
   , defaultLMDBLimits -- TODO this is just for convenience, should remove
+  -- * Required for db-checker
+  , LmdbBox(..)
```

Comment:
This will be removed if https://github.com/input-output-hk/ouroboros-network/pull/3756 get's merged